pkgs/development/libraries/science/math/cudnn/extension.nix

```diff
@@ -15,6 +15,8 @@ final: prev: let
 
   majorMinorPatch = str: lib.concatStringsSep "." (lib.take 3 (lib.splitVersion str));
 
+  major = str: lib.concatStringsSep "." (lib.take 1 (lib.splitVersion str));
```

Comment:
Can this use lib.versions.major/majorMinor?

Reply:
yes, I'll change it for simplicity